apps/sample-next-auth-app/README.md

```diff
@@ -1,60 +1,60 @@
 # Xata + Next.js + Auth.js
 
```

Comment:
It might just be me, and feel free to disagree, but I always appreciate an image of the output that this sample will have. So a screenshot, gif or video of what this produces or what the app looks like.

Reply:
Definitely agree. We might want to take a design pass at this since it resembles our old branding. 

It's a very easy & simple app (literally just a sign in and signed-in view of a landing page)

Reply:
Generally, I think the screenshots are great for contrived examples of real-world scenarios. But in this case it's a starter (the minimum amount of bells and whistles, it's like a blank template) so I'd say there isn't much point in adding a screenshot - it's just another thing for the user to delete from `/public` and doesn't really offer much value.

Reply:
I don't think it's _too_ much of a burden to delete an image from `public` if we have a good reason to add it. 

I wouldn't mind adding a gif showing the auth process to the signed-in dashboard, but I'm open to opinions there. It is a _very_ simple example with not much to show, but that might be perfect for a gif at the top of the README?

Reply:
Sure. I don't mind.
I didn't mean it as a burden, more that it adds little to no value. But I don't have a strong opinion on it.